ecs - one line install script (#106)
amonks/run: README.md

# Installation

Run is a single binary, which you can download from from the [releases page][releases].

## Install Script (MacOS, Linux)

> Download and install the latest version of `run` to the current working directory.

Reviewer: this reads like an instruction, when, in fact, it is a description of the install script

> Download and install the latest version of `run` to the current working directory.

Reviewer: this reads a bit like an instruction, rather than a description of the install script. Please make it clearer.

(one idea: a prefix like "this command will…")

> This command retrieves and executes the shell installation script for the latest version of run, installing it into the current working directory.

Author: ```suggestion
This command can be used to fetch the install script and execute it in the current working directory.
```

Author: I'm not sure which of these is better?

Reviewer: "fetching the script" feels like an implementation detail to me--it's confusing for casual readers, and already obvious to sophisticated ones

how about, "This install command will place download the latest version of run for your platform and place it in the current directory"
